includes/discovery/sensors/state/vrp.inc.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * vrp.inc.php
+ *
+ * LibreNMS sensors state discovery module for HP Procurve
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 PipoCanaja
+ * @author     PipoCanaja
+ */
+
+//d_echo($pre_cache);
```

Comment:
Looks good, you just need to remove this commented debug code :)

Reply:
missed them :( done now :D